paddle/fluid/framework/threadpool.cc

```diff
@@ -34,6 +34,11 @@ ThreadPool* ThreadPool::GetInstance() {
   return threadpool_.get();
 }
 
+void ThreadPool::Reset() {
```

Comment:
Does the `Reset()` method can be called on everywhere? This may be caused to multiple instances such as:
``` c++
thread1: t1 = GetInstance(); t->Run(f);  ThreadPool::Reset();  t2 = GetInstance(); t2->Run(f)
thread2: t3 = GetInstance(); t->Run(f1); t->Run(f2) ...
```

There could be two ThreadPool instances in the program, t3 in thread2 is not the same instance with t2 in thread1.

Reply:
clarified. It's only used by test.